homeassistant/components/deconz/gateway.py

```diff
@@ -76,10 +78,14 @@ def __init__(
         self.deconz_ids: dict[str, str] = {}
         self.entities: dict[str, set[str]] = {}
         self.events: list[DeconzAlarmEvent | DeconzEvent] = []
+        self.ignored_devices: set[tuple[Callable[[EventType, str], None], str]] = set()
 
         self._option_allow_deconz_groups = self.config_entry.options.get(
             CONF_ALLOW_DECONZ_GROUPS, DEFAULT_ALLOW_DECONZ_GROUPS
         )
+        self.option_allow_new_devices = self.config_entry.options.get(
+            CONF_ALLOW_NEW_DEVICES, DEFAULT_ALLOW_NEW_DEVICES
+        )
```

Comment:
I just noticed this one: Why is this part of the options flow?
Allowing/disallowing new devices is already a higher-level option, right? As in, for ANY integration?

![CleanShot 2022-05-16 at 23 38 52](https://user-images.githubusercontent.com/195327/168686915-3fc4de6a-a9fe-47de-af34-6e8e41dc040d.png)

Reply:
The Main difference is that the system option creates the entry in entity registry settling on the entity id and name. This options allow the user take control and prepare naming in deconz prior to it being added. So they're different.

Reply:
Hehe ok, It doesn't matter for this PR that much, it's just that I noticed... And I'm struggling with the difference (which I think a user might too?).

Anyways, thanks for clarifying 👍